UefiPayloadPkg/UefiPayloadPkg.dsc

```diff
@@ -94,6 +94,7 @@
   DEFINE LOAD_OPTION_ROMS               = TRUE
   DEFINE SECURE_BOOT_DEFAULT_ENABLE     = TRUE
   DEFINE DASHARO_SYSTEM_FEATURES_ENABLE = FALSE
+  DEFINE SETUP_PASSWORD_ENABLE          = FALSE

```

Comment:
Remove it, we actually want to have a password compiled by default, as it is a standard feature.

Reply:
Ok, removed the option entirely